crate2nix/src/lib.rs

```diff
@@ -152,11 +152,6 @@ fn prefetch_and_fill_crates_sha256(
     }) {
         if let Some(hash) = lock_file.get_hash(&package.package_id.repr)? {
             package.source = package.source.with_sha256(hash);
-        } else {
```

Comment:
Why did you remove this? I'd like to include it so that it is clear when this is happening if the Cargo.lock format should change.

Reply:
It seemed a bit noisy to me. Would you be open to a PR that adds logging with the ability to choose at runtime the level, so that say this message would only show up at the `Debug` level?

Reply:
Happy to bring this back and open up another PR to add logging.

Reply:
Ok, brought it back.